auth/resources/authserver.properties.sample

```diff
@@ -72,3 +72,10 @@ cacheTimeout=30
 
 # IEM Server details for sending Auth IEM alerts,
 IEMServerURL=http://127.0.0.1:28300/EventMessage/event
+
+# Rsyslog server details for sending IAM audit log,
```

Comment:
Please add them in unsafe attribute

Reply:
No required. As per Discussion with Ujjwal, Host and Port may change in future. so should be upgradable